IaC/pmm.cd/init.groovy.d/matrix.groovy

```diff
@@ -148,6 +148,7 @@ authz_strategy_config = [
         'percona*PMM': ['Overall Read','Agent Build','Agent Configure','Agent Connect','Agent Create','Agent Delete','Agent Disconnect','Job Build','Job Cancel','Job Configure','Job Create','Job Delete','Job Discover','Job Move','Job Read','Job Workspace','Run Delete','Run Update','Run Replay','View Configure','View Create','View Delete','View Read','SCM Tag'],
         'percona*ext-victoriametrics': ['Overall Read','Job Build','Job Cancel','Job Read','Run Replay','Run Update'],
```

Comment:
`percona*ext-victoriametrics` can be deleted

Reply:
@atymchuk Please re-review, updated in  ecb8fbd